core/src/main/java/io/grpc/ServerBuilder.java

```diff
@@ -41,6 +41,11 @@
  * @param <T> The concrete type of this builder.
  */
 public abstract class ServerBuilder<T extends ServerBuilder<T>> {
+
+  public static ServerBuilder<? extends ServerBuilder<?>> forAddress(String name, int port) {
```

Comment:
Hmm... I thought we were discussing just having port: `forPort(int port)`. We expect in the future we will have something like `forName(String name)`, but in the mean time only letting binding to a port seems quite reasonable. With the current `forAddress()` here we would need to tell users how to listen on all interfaces for the given port.